packages/liquidator/test/Liquidator.js

```diff
@@ -750,17 +745,8 @@ contract("Liquidator.js", function(accounts) {
             async function() {
               let errorThrown;
               try {
-                liquidatorConfig = {
-                  ...liquidatorConfig,
-                  crThreshold: -0.02
-                };
+                liquidatorConfig = { ...liquidatorConfig, crThreshold: -0.02 };
```

Comment:
Is the small buffer for liquidations mainly to be nice to people with very slightly undercollateralized positions, to avoid DVM votes that might have ambiguous outcomes, or both?

Reply:
yup. note this is just a unit test but that is the idea.

the changes in this file were purely cosmetic.